File: PHP/uberVan.php
seatsMaterial
typeCarAccepted

<?php

use function PHPSTORM_META\type;

require_once("car.php");

class uberVan extends Car{
    public $seasMaterial;
    public $typeCarAccepted;

    public function __construct($driver, $license, $typeCarAccepted,$seasMaterial){
        parent::__construct($license, $driver);
        $this->driver = $driver;
        $this->license = $license;
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seasMaterial = $seasMaterial;
    }
}


?>

File: Python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []
    def __init__(self, driver, license, typeCarAccepted, seatsMaterial):
        super().__init__(driver, license)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial